src/darwin/Framework/CHIP/MTRMetricKeys.h

```diff
@@ -0,0 +1,48 @@
+/**
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <tracing/metric_event.h>
+
+constexpr chip::Tracing::MetricKey kMetricDarwinFrameworkDeviceCommissioning = "dwnfw_device_commissioning";
```

Comment:
We could put these into a C++ namespace instead of using these super long names?
With code like

```
VerifyOrReturnValueWithMetric(kMetricDarwinFrameworkStartBrowseForCommissionables, ...)
```

it's starting to get hard to see the actual code.

Reply:
I could see what that looks like